.gitignore

```diff
@@ -1,5 +1,6 @@
 # Ignore bundler config.
 /.bundle
+vendor/*
```

Comment:
Not clear if this was needed at some point or it just slipped into the repo.
Unit tests and manual cypress tests still pass after this was removed.

Also we had this path under .cfignore, so it never made it to the PaaS, which build everything from scratch.